net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditOSPFNetwork.xml

```diff
@@ -20,6 +20,12 @@
     <type>text</type>
     <help>Area in wildcard mask style like 0.0.0.0 and no decimal 0</help>
   </field>
+  <field>
+    <id>network.arearange</id>
+    <label>Area Range</label>
```

Comment:
Range sounds misleading. Is this a Quagga/FRR term?

Reply:
Yep

Reply:
if you look at the config, then probably yes:

```
area {{ network.area }} range {{ network.arearange }}
```

Reply:
I think there are more ways to achieve route summarization .. thats why range would fit better